build.gradle

```diff
@@ -16,6 +16,10 @@ plugins {
 
 repositories {
     mavenCentral()
+
+    maven {
+        url "https://artifactory.broadinstitute.org/artifactory/libs-snapshot/" // for intel deflater snapshots
```

Comment:
`gatk-native-bindings` is the set of interfaces that GATK uses/will use to interact with native code in general (not just the PairHMM). It is super-lightweight (just interfaces, and perhaps eventually a very small number of utilities). I do agree that it is a bit awkward to have htsjdk depend on a project with gatk in the name, even transitively -- we are considering switching to a more generic name.